src/wazuh_modules/task_manager/wm_task_manager_db.c

```diff
@@ -149,10 +152,6 @@ int wm_task_manager_get_task_status(int agent_id, const char *module, char **sta
     int result = 0;
     int task_id = OS_INVALID;
 
-    if (sqlite3_open_v2(TASKS_DB, &db, SQLITE_OPEN_READWRITE, NULL)) {
```

Comment:
Why did you remove this?

Reply:
Shouldn't have deleted it, error in the Merge